extensions/typescript-language-features/src/features/completions.ts

```diff
@@ -607,8 +610,12 @@ function shouldExcludeCompletionEntry(
 	element: Proto.CompletionEntry,
 	completionConfiguration: CompletionConfiguration
 ) {
+	const isNameSuggestion = element.kind === PConst.Kind.warning
```

Comment:
Needs a semicolon 😬

(agree with the refactoring here btw, much more readable 👍)

Reply:
Ah, right. Reflexes 😄